uqcsbot/scripts/hoogle.py

```diff
@@ -0,0 +1,57 @@
+from uqcsbot import bot, Command
+import requests
+import json
+import html
+
+def get_endpoint(type_sig: str) -> str:
+    unescaped = html.unescape(type_sig)
+
+    return "https://www.haskell.org/hoogle/?mode=json&hoogle=" + unescaped + "&start=0&count=10"
+
+def pretty_hoogle_result(result: dict, is_verbose: bool) -> str:
+    url = result['location']
+    type_sig = result['self']
+    docs = result['docs']
+
+    if is_verbose:
+        return f"`{type_sig}` <{url}|link> {docs}"
```

Comment:
Formatting  could definitely be better for verbose mode

Reply:
`\n` between the link and the docs?

Reply:
throw up a screenshot of both options?

Reply:
(so we can see + help decide)

Reply:
Added screenshots

Reply:
`\n` lgtm